### PR TITLE
David

### PR DESCRIPTION
**Pull request by davyuu**
Not sure how to not include account info in request. Probably should move account info to separate file for easier pull requests.

### CODE REVIEW

File: util/commands.js [hunk 1]
```diff
@@ -11,12 +14,52 @@ var getEvent = function(param) {
...
+      resp.message(param + ' team is not found');
+  }
+  return resp.toString();
+}
+
+var addPoints = function(param) {
+  console.log('adding points', param);
+  resp = new twilio.TwimlResponse();
+  var obj = JSON.parse(fs.readFileSync(pointsFile, 'utf8'));
+  if(param.length = 2){
```
Comment (jstarw) [inline util/commands.js]: should this be '==' instead of '='?

File: util/util.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 var twilio = require('twilio');
 var commands = require('./commands.js')
 
-var ACCOUNT_SID = 'ACd23638c1417ac036b6ba46b9189f8cf2';
-var AUTH_TOKEN = '72c4891dc9fb486dff01fa40f563a764';
-var PHONE_NUMBER = '+16474901601'
+var ACCOUNT_SID = 'ACff97a9e8e18e7ea7d4a6cf80b173513c';
+var AUTH_TOKEN = '580dade8c27e471ae44a9ef91abd45a7';
+var PHONE_NUMBER = '+16474926522'
```
Comment (jstarw) [inline util/util.js]: I think we should not commit these account info changes, just keep it locally

File: data/Schedule.csv [hunk 3]
```diff
@@ -0,0 +1,54 @@
+,WEDNESDAY,THURSDAY,FRIDAY,SATURDAY
+9:00 AM,,,,
```
Comment (jstarw) [inline data/Schedule.csv]: for the times, should we store them as a different format? (UTC maybe?)

**Comment by davyuu** [comment]
For the account stuff, I think we can create a separate file for the info so we can exclude that file from commits

**Comment by jstarw** [comment]
Yeah we can put them into a config file. Let me know when you finish addressing these issues.

File: util/commands.js [hunk 4]
```diff
@@ -11,12 +14,52 @@ var getEvent = function(param) {
...
+          resp.message('added ' + points + ' points to ' + team + ' team, ' + team + ' team now has ' + newPoints + ' points');
+      }
+  }
+  else{
+      resp.message('wrong format, should be: addpoints [team] [points]');
+  }
   return resp.toString();
 }
 
+function replacer(key, value){
```
Comment (jstarw) [inline util/commands.js]: What is this function for?

**Comment by jstarw** [comment]
David dont worry about the account stuff, i moved them into a config file in my branch.

**Pull request merged**